content/en/aws/docdb/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "DocumentDB (DocDB)"
+linkTitle: "DocumentDB (DocDB)"
+categories: ["LocalStack Pro"]
+description: >
+  DocumentDB (DocDB)
+---
+
+LocalStack supports a basic version of [Amazon DocumentDB](https://aws.amazon.com/documentdb/) for testing.
+
+When loading the DocumentDB service, LocalStack will create a MongoDB container to handle DocumentDB storage. By default, this instance will listen on a port in the 33000-34000 port range and will instantiate a master user named 'test' without a password. The `mongosh` client may be used to log in to and interact with the instance.
+
+{{< alert title="Note" >}}
+Because LocalStack utilizes a MongoDB container to provide DocumentDB storage, LocalStack may not have feature parity with Amazon DocumentDB. The database engine may support additional features that Amazon DocumentDB does not and vice versa.
+{{< /alert >}}
+
+## Example
+
+{{< command >}}
+$ awslocal docdb create-db-cluster --db-cluster-identifier 'test-docdb-cluster' --engine 'docdb'
+{
+    "DBCluster": {
+        "DBClusterIdentifier": "test-docdb-cluster",
+        "DBClusterParameterGroup": "default.docdb",
+        "Status": "creating",
+        "Endpoint": "localhost:33101",
+        "MultiAZ": false,
+        "Engine": "docdb",
+        "Port": 33101,
+        "MasterUsername": "test",
+        "StorageEncrypted": false,
+        "DBClusterArn": "arn:aws:rds:us-east-1:000000000000:cluster:test-docdb-cluster"
+    }
+}
+
+
+$ awslocal docdb create-db-instance --db-instance-identifier 'test-docdb-instance' --db-instance-class 'db.r5.large' --engine 'docdb' --db-cluster-identifier 'test-docdb-cluster'
```

Comment:
It would also be good to add a note about the DB instance and how we essentially mock this part and isn't exactly demonstrative of the real DB instance class.